File: Variables/Expressions/budgetseven.c
#include <stdio.h>

int main(void) {
    float income, rent, utilities, groceries, transportation, expenses, savings, total;
    printf("This is going to calculate your budget for the month.");
    printf("How much do you make in a month?\n");
    scanf("%f", &income);
    scanf("Your income is: $%.2f\n", income);
    printf("How much is rent?\n");
    scanf("%f",&rent)
    printf("How much is utilities?\n");
    scanf("%f",&utilities);
    printf("How much is groceries?\n");
    scanf("%f", &groceries);
    printf("How much is transportation\n");
    scanf("%f", &transportation);
    expenses = rent + utilities + groceries + transportation 
    savings = income *.2;
    total = income - expenses - savings;
    printf("Your income is: $%.2f\n", income);
    printf("Your expenses are: $%.2f\n", expenses);
    printf("Your savings are: $%.2f\n", saving); 
    printf("Your total left to spend is: $%.2f\n", total);
    return 0;
}

File: Budget/budgetseven.py
print("This is going to calculate your budget for the month.")
income = float (input("What is your monthly income: "))
rent = float (input("What is your monthly rent: "))
utilities = float (input("What is your monthly utilities: "))
groceries = float (input("What is your monthly groceries: "))
transportation = float (input("What is your monthly transportation: "))
expenses = rent + utilities + groceries + transportation
savings = income * 0.2
total = income - expenses - savings
prent = rent/income * 100
putilities = utilities/income *100
ptransportation = transportation/income * 100
pexpenses = expenses/income * 100
print("Your income is: %\n", income)
print("Your expenses are: %\n", expenses)
print("Your savings are: %\n", savings)
print("Your total left to spend is: $%.2f\n", total)
print("Your rent is % of your income", prent)
print("Your rent is % of your income", putilities) 
print("Your rent is % of your income", ptransportation)